pom.xml

```diff
@@ -137,12 +137,12 @@
 	</modules>
 
 	<properties>
-		<spring-cloud-stream.version>1.2.0.RC1</spring-cloud-stream.version>
-		<spring-cloud-deployer-spi.version>1.2.0.M3</spring-cloud-deployer-spi.version>
-		<spring-cloud-deployer-local.version>1.2.0.M3</spring-cloud-deployer-local.version>
-		<spring-cloud-stream-binder-rabbit.version>1.2.0.RC1</spring-cloud-stream-binder-rabbit.version>
-		<spring-cloud-deployer-resource-support.version>1.2.0.M3</spring-cloud-deployer-resource-support.version>
-		<spring-cloud-deployer-resource-maven.version>1.2.0.M3</spring-cloud-deployer-resource-maven.version>
+		<spring-cloud-stream.version>1.2.0.RELEASE</spring-cloud-stream.version>
```

Comment:
as a quick note - it would be better to use the SCSt BOM for centrally managing deps https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-stream-dependencies/Chelsea.SR1/spring-cloud-stream-dependencies-Chelsea.SR1.pom